docs/src/main/tut/datatypes/ior.md

```diff
@@ -98,22 +98,23 @@ To extract the values, we can use the `fold` method, which expects a function fo
 ```tut
 
 validateUser("john.doe", "password").fold(
-  errorNel => s"Error: ${errorNel.head}",
+  errorNec => s"Error: ${errorNec.head}",
   user => s"Success: $user",
   (warnings, user) => s"Warning: ${user.name.value}; The following warnings occurred: ${warnings.show}"
 )
 
 ```
-Similar to [Validated](validated.html), there is also a type alias for using a `NonEmptyList` on the left side.
+Similar to [Validated](validated.html), there is also a type alias for using a `NonEmptyChain` on the left side.
 
 ```tut:silent
-type IorNel[B, A] = Ior[NonEmptyList[B], A]
+type IorNec[B, A] = Ior[NonEmptyChain[B], A]
 ```
 
 
 ```tut
+import cats.implicits._, cats.data.NonEmptyChain
 
-val left: IorNel[String, Int] = Ior.leftNel("Error")
+val left: IorNec[String, Int] = Ior.fromEither("Error".leftNec[Int])
```

Comment:
Seems I missed the convenience function here, we should add an Issue for that too. :)